clients/docs/java.rst

```diff
@@ -0,0 +1,320 @@
+   code|clients/cloud/java/src/main/java/io/confluent/examples/clients/cloud/ProducerExample.java`.
+
+
+Consume Records
+~~~~~~~~~~~~~~~
+
+#. Run the consumer, passing in arguments for:
+
+   - the local file with configuration parameters to connect to your Kafka cluster
+   - the topic name you used earlier
+
+   .. code-block:: text
+
+      # Compile the Java code
+      mvn clean package
+
+      # Run the consumer
+      mvn exec:java -Dexec.mainClass="io.confluent.examples.clients.cloud.ConsumerExample" \
+      -Dexec.args="$HOME/.confluent/java.config test1"
+
+#. Verify the consumer received all the messages. You should see:
+
+   .. code-block:: bash
+
+      ...
+      Consumed record with key alice and value {"count":0}, and updated total count to 0
+      Consumed record with key alice and value {"count":1}, and updated total count to 1
+      Consumed record with key alice and value {"count":2}, and updated total count to 3
+      Consumed record with key alice and value {"count":3}, and updated total count to 6
+      Consumed record with key alice and value {"count":4}, and updated total count to 10
+      Consumed record with key alice and value {"count":5}, and updated total count to 15
+      Consumed record with key alice and value {"count":6}, and updated total count to 21
+      Consumed record with key alice and value {"count":7}, and updated total count to 28
+      Consumed record with key alice and value {"count":8}, and updated total count to 36
+      Consumed record with key alice and value {"count":9}, and updated total count to 45
+
+#. When you are done, press ``Ctrl-C``.
+
+
+Kafka Streams
+-------------
+
+#. Run the Kafka Streams application, passing in arguments for:
+
+   - the local file with configuration parameters to connect to your Kafka
+     cluster
+   - the topic name you used earlier
+
+   .. code-block:: text
+
+      # Compile the Java code
+      mvn clean package
+
+      # Run the Kafka streams application
+      mvn exec:java -Dexec.mainClass="io.confluent.examples.clients.cloud.StreamsExample" \
+      -Dexec.args="$HOME/.confluent/java.config test1"
+
+   You should see:
+
+   .. code-block:: bash
+
+      ...
+      [Consumed record]: alice, 0
+      [Consumed record]: alice, 1
+      [Consumed record]: alice, 2
+      [Consumed record]: alice, 3
+      [Consumed record]: alice, 4
+      [Consumed record]: alice, 5
+      [Consumed record]: alice, 6
+      [Consumed record]: alice, 7
+      [Consumed record]: alice, 8
+      [Consumed record]: alice, 9
+      ...
+      [Running count]: alice, 0
+      [Running count]: alice, 1
+      [Running count]: alice, 3
+      [Running count]: alice, 6
+      [Running count]: alice, 10
+      [Running count]: alice, 15
+      [Running count]: alice, 21
+      [Running count]: alice, 28
+      [Running count]: alice, 36
+      [Running count]: alice, 45
+      ...
+
+#. When you are done, press ``Ctrl-C``.
+
+#. View the :devx-examples:`consumer
```

Comment:
Unnecessary line break?

Reply:
Oh sorry —just a styling preference on my end. I can remove that. :-)

Reply:
Updated